File: app/src/main/java/io/github/ovso/blackbox/utils/DateUtils.java
package io.github.ovso.blackbox.utils;

import java.text.SimpleDateFormat;
import java.util.Date;

public final class DateUtils {

  private DateUtils() {
  }

  public static String getDate(Date date, String format) {
    SimpleDateFormat simpleDateFormat = new SimpleDateFormat(format);
    return simpleDateFormat.format(date);
  }
}

File: app/src/main/java/io/github/ovso/blackbox/utils/DateUtils.kt
package io.github.ovso.blackbox.utils

import java.text.SimpleDateFormat
import java.util.Date

object DateUtils {

  fun getDate(
    date: Date,
    format: String
  ): String {
    val simpleDateFormat = SimpleDateFormat(format)
    return simpleDateFormat.format(date)
  }
}